test/upload-download.test.js

```diff
@@ -191,7 +191,7 @@ describe('storage service', function() {
       .set('Connection', 'keep-alive')
       .expect('Content-Type', /json/)
       .expect(500, function(err, res) {
-        assert(res.body.error.message.indexOf('is not allowed') !== -1);
+        assert(res.body.error);
```

Comment:
This change is a step backwards. Before, the test was asserting that the request fails with a specific error ("is not allowed"). Now the tests checks that the test fails with any Internal Server Error.

Please change the way how the `app` is setup, so that the strong-error-handler middleware is in `debug` mode and the error message includes full details.

I think you need to disable the built-in error handler in strong-error-handler and then explicitly add a new strong-error-handler middleware after `loopback.rest()`. Please look up more information in the documentation and let me know if you need any help.